third_party/cc/gtest.build

```diff
@@ -0,0 +1,9 @@
+cc_library(
+    name = "gtest",
+    srcs = glob(["googletest/src/*.cc"], exclude=["googletest/src/gtest-all.cc"]),
+    hdrs = glob(["googletest/include/**.h","googletest/src/*.h"]),
```

Comment:
Think this should be
```
hdrs = glob(["googletest/include/**.h"]),
private_hdrs = glob(["googletest/src/*.h"]),
```
It's a little bit purist but the headers in `src` presumably aren't meant to be consumed by other things.